src/js/header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './css/header.css';
+
+const Header = () => (
+  <header>
+    <h1>Edith Magic</h1>
+    <ul>
+      <a href="/">Home</a>
+      <a href="/Calculator">Calculator</a>
+      <a href="/Quotes">Quotes</a>
+    </ul>
+  </header>
+);
+
+export default Header;
```

Comment:
Great job @vigehi 🤝

- [ ] Kindly maintain one `header.js` file. Your project appears to be having two similar files. You may remove one and keep the other.

Reply:
Well resolved 👍🏽